src/main/webapp/resources/js/pages/sequence-files/components/FastQCDetails.jsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * This file renders the FastQC details component.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Divider, Layout, Typography } from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import { grey1 } from "../../../styles/colors";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { BasicList } from "../../../components/lists";
+import { getFastQCDetails } from "../../../apis/files/sequence-files";
+import { formatDate } from "../../../utilities/date-utilities";
+import { ContentLoading } from "../../../components/loader";
+import { seqFileId, seqObjId } from "../fastqc-constants";
+
```

Comment:
Comments

Reply:
Added in 139845ae0399cc73ea9aa42936a8341d24590480